lms/urls.py

```diff
@@ -613,6 +613,17 @@
         url(r'^certificates/html', 'certificates.views.render_html_view', name='cert_html_view'),
     )
 
+# Access to courseware as an LTI provider
+if settings.FEATURES["ENABLE_LTI_PROVIDER"]:
+    urlpatterns += (
+        url(r'^courses/{}/lti_launch/(?P<chapter>[^/]*)/(?P<section>[^/]*)/(?P<position>[^/]*)/?$'.format(settings.COURSE_ID_PATTERN),
+            'lti_provider.views.lti_launch', name="lti_provider_launch"),
+    )
+    urlpatterns += (
+        url(r'^lti_provider/lti_run/',
+            'lti_provider.views.lti_run', name="lti_provider_run"),
+    )
```

Comment:
I would strongly suggest defining a list of API urls within your app, and then just doing a one-line include of all those endpoints in this file. It's a lot cleaner, and it makes it much clearer exactly what is being enabled by setting the feature flag.

See https://github.com/mcgachey/edx-platform/blob/mcgachey-lti-provider/lms/urls.py#L348 for an example, and lms/djangoapps/instructor/views/api_urls.py for the file where that app's urls are defined.